### PR TITLE
Vet Clinic Database Milestone: Query multiple tables

### PR DESCRIPTION
**Pull request by CarlosIgreda**
For this Milestone I:

✅ Used GitFlow
✅ Used primary key & foreign key mechanisms for joining tables
✅ Queried multiple tables
✅ Prepared complex queries that answer analytical questions

Screenshots:

✅ What animals belong to Melody Pond?

image

✅ List of all animals that are pokemon (their type is Pokemon)

image

✅ List all owners and their animals, remember to include those that don't own any animal

image

✅ How many animals are there per species?

image

✅ List all Digimon owned by Jennifer Orwell

image

✅ List all animals owned by Dean Winchester that haven't tried to escape

image

✅ Who owns the most animals?

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ I will use a relational database to create the data structure for a vet clinic.
 ### Key Features <a name="key-features"></a>
 
 - **Follow a correct Gitflow**
-- **Use database transactions**
-- **Modify and delete data in SQL**
+- **Use primary key & foreign key mechanism for joining tables**
+- **Query multiple tables**
```
Comment (abdulhamiid) [inline README.md:43]: image

image

Kindly fix the broken license link, please let's avoid broken links in our readme for a professional repository. 🤗🛠

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @CarlosIgreda 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Use primary key & foreign key mechanisms for joining tables.
- ✔️ Query multiple tables.
- ✔️ Prepare complex queries that answer analytical questions.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CarlosIgreda) [reply README.md:43]: @abdulhamiid Done it.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @CarlosIgreda       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  a. 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] No linter errors
- [x] App Works as intended 💯


 
 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Comment by CarlosIgreda** [comment]
@KayLemba Thank you buddy for your feedback! 🚀

**Pull request merged**